tools/screenshot-test/src/app/firebase.service.ts

```diff
@@ -71,10 +71,7 @@ export class FirebaseService {
             this._readResults(childSnapshot);
             break;
         }
-        counter++;
-        if (counter === snapshot.numChildren()) {
-          return true;
-        }
+        return ++counter === snapshot.numChildren();
```

Comment:
I prefer not to use increment/decrement expressions as part of a larger expression where the order of the increment/decrement would matter
(here and elsewhere)